### ISSUE TITLE
Benchmarking dependencies and references

### ISSUE THREAD

**Issue by i64** [Battery-Intelligence-Lab/DTWpp#13]
Hi authors, I am reviewing the code from openjournals/joss-reviews#6881

- No "requirments.txt" (or something equivalent) is provided to run the python benchmarks; however, they use third-party libraries.
- In the [bibliography](https://github.com/Battery-Intelligence-Lab/dtw-cpp/blob/4e3b51ef5b6bce9ccbc6690cb755bfbb0350d548/joss/paper.bib#L46C18-L46C30), the library "dtaidistance" ([github](https://github.com/wannesm/dtaidistance/releases), [zenodo](https://zenodo.org/records/5901139)) references an old version from 2020 (v2.0.0), while there is a more modern version from 2022(v2.3.10). Is the 2020 version used mistakenly?

**Comment by ElektrikAkar** [comment]
Hi i64,

Thank you very much for your comments!

- Yes, we have now added a [requirements.txt](https://github.com/Battery-Intelligence-Lab/dtw-cpp/blob/main/benchmark/requirements.txt) for benchmarks. However, I should note that the `TSlearn` and `dtaidistance` libraries require different `numpy` versions. Therefore, we wrote the [requirements.txt](https://github.com/Battery-Intelligence-Lab/dtw-cpp/blob/main/benchmark/requirements.txt) considering `dtaidistance` as we only presented results regarding this library in the paper.
- You are right that we referred to an older version of the `dtaidistance` library. I think we copied an older URL from the repository of the software. Checking the version we used, we see that it was version `2.3.10`; therefore, we updated the reference accordingly (which is also reflected in the [requirements.txt](https://github.com/Battery-Intelligence-Lab/dtw-cpp/blob/main/benchmark/requirements.txt)).

See 4750f5837be4a97807b2e69cf92e98f66887fcfe for the details.
